CMakeLists.txt

```diff
@@ -183,7 +183,7 @@ string(REGEX MATCH "(^|;)[Ss]ubmodule(^|;)" INCLUDE_INDIRECT_DEPS "${HYBRID_CONF
 
 if(WITH_MPI)
   set(MPIFX_GIT_REPOSITORY "https://github.com/dftbplus/mpifx.git")
-  set(MPIFX_GIT_TAG "da51073aa87831a91ae756a9773e39ea000d1c3a")  # do not change manually!
+  set(MPIFX_GIT_TAG "0641eead6ff9bf04c7ca2ec676a281780e2029e2")  # do not change manually!
```

Comment:
This is behind the main branch dependency (from before the rebase time).

Reply:
Fixed!